addons/binding/org.openhab.binding.tellstick/README.md

```diff
@@ -42,16 +42,12 @@ If you want to use the *Telldus Live* its bridge, *Telldus Live bridge* need to
 
 ## Binding Configuration
 
-***For USB connected tellsticks only, eg. Basic and DUO***
-First of all you need to make sure that your JVM is matching your installed Telldus Center.
-This normally means openHab must run on a 32bit JVM for windows and a 64bit JVM for linux.
-For windows the binding is hardcoded to look for Telldus Center in Programs Files ("C:/Program Files/Telldus/;C:/Program Files (x86)/Telldus/").
-If you have trouble getting the telldus core library to work you can modify the library path using
+The binding in itself require no configuration.
 
 ## Thing Configuration
 
 Only the bridges require manual configuration.
-The devices and sensors should not be added by hand, let the discovery/inbox initially configure these.
+Devices and sensors are preferable discovered automatically, let the discovery/inbox initially configure these. Add them either with karaf: `inbox approve <thingId>` or in Paper UI.
```

Comment:
Consider, "It is preferable that devices and sensors are discovered automatically; let the discovery/inbox initially configure them."  "You can add them either with karaf: `inbox approve <thingId>` or by using the inbox of the PaperUI."